iroh/src/rpc_protocol.rs

```diff
@@ -256,6 +269,333 @@ pub struct VersionResponse {
     pub version: String,
 }
 
+// peer
+
+/// todo
```

Comment:
lets either fill in the `todo`s or remove them

Reply:
removed them for now